source/DSCResources/DSC_CMSiteMaintenance/en-US/DSC_CMSiteMaintenance.strings.psd1

```diff
@@ -0,0 +1,14 @@
+ConvertFrom-StringData @'
+    RetrieveSettingValue = Getting results for Configuration Manager maintenance task.
+    TestState            = Test-TargetResource compliance check returned: {0}.
+    TestDisabled         = {0} currently is enabled and should be disabled.
+    Days                 = {0} expected {1} to be selected.
+    MissingBackupLoc     = No backuplocation specified and no default value exists.
```

Comment:
I would either put a space between backup location or capitalize like the param.